oneview_redfish_toolkit/api/storage.py

```diff
@@ -92,9 +92,12 @@ def __init__(self,
             sorted(self.map_supported_device_protos(drive_technologies))
         self.redfish["StorageControllers"].append(storage_controllers)
 
-        count_drives_by_jbod = \
-            [int(item["numPhysicalDrives"]) for item in sas_logical_jbods]
-        count_drives = reduce(operator.add, count_drives_by_jbod)
+        count_drives = 0
+        if len(sas_logical_jbods) > 0:
+            count_drives_by_jbod = \
+                [int(item["numPhysicalDrives"]) for item in sas_logical_jbods]
+
+            count_drives = reduce(operator.add, count_drives_by_jbod)
```

Comment:
Just a tip, it is not a necessary change, we could simply start the initial value of `reduce` method with zero: `reduce(operator.add, count_drives_by_jbod, 0)`, so the `count_drives` will be `0` when the `sas_logical_jbods` is empty.